signup_api/user.py

```diff
@@ -0,0 +1,97 @@
+from datetime import datetime as dt
+
+
+class User:
+    def __init__(self, obj_repo):
+        self.repo = obj_repo
+
+    def set_user_info(self, usr_dta):
+        """ Method to set user information. """
+        try:
+            claims = usr_dta['authorizer']['claims']
+            email_verified = claims['email_verified']
+            status = 0
+            if email_verified:
+                status = 1
+            else:
+                raise Exception("Email Verification is pending.")
+            q_dta = [claims['cognito:username'], usr_dta['accountId'], claims['name'], claims['email'], status,
+                     status, usr_dta['requestId'], usr_dta['requestTimeEpoch'], dt.utcnow(), dt.utcnow()]
+            set_usr_dta = self.repo.execute(
+                "INSERT INTO user (username, account_id, name, email, email_verified, status, request_id, "
+                "request_time_epoch, row_created, row_updated) "
+                "VALUES(%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)", q_dta)
+            if len(set_usr_dta) > 0:
```

Comment:
In what circumstance will len(set_usr_dta) be 0? The insert will either succeed or run into a constraint exception in which case you will get an exception right?